components/MentorshipsList/index.js

```diff
@@ -0,0 +1,46 @@
+import { Alert, AlertIcon } from "@chakra-ui/alert";
+import { SimpleGrid, Text } from "@chakra-ui/layout";
+import useSWR from "swr";
+import MentorshipCard from "./MentorshipCard";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+const MentorshipsList = () => {
+  const { data, error } = useSWR("/api/mentorships", fetcher);
+  if (error) {
+    return (
+      <Alert mt={10} status="error">
+        <AlertIcon />
+        Error retrieving the mentorships list. {error.toString()}
+      </Alert>
+    );
+  }
+
+  if (!data) {
+    return <div>Loading..</div>;
+  }
+
+  if (data.records.length == 0) {
+    return (
+      <Text mt={10} fontWeight={700}>
+        You don&apos;t have any active mentorship going on. We will notify you when
+        someone requests you as a mentor.
```

Comment:
This is the mentee list, right, so it should say something like "You can <a>Find a Mentor</a>" or similar.

Reply:
ah right, I switched the sentences with the ones for the mentor's mentees list. Nice finding!